build.gradle

```diff
@@ -204,6 +204,11 @@ configure(subprojects.findAll { !javaPlatformModules.contains(it.name) }) {
     }
   }
 
+  configurations {
+    all*.exclude group: 'javax.servlet', module: 'javax.servlet-api' // prefer jakarta.servlet-api
+    all*.exclude group: 'commons-logging', module: 'commons-logging' // prefer jcl-over-slf4j
```

Comment:
Yes. It will add the following exclusions to **all** dependencies in the pom:
```
 <exclusions>
        <exclusion>
          <artifactId>javax.servlet-api</artifactId>
          <groupId>javax.servlet</groupId>
        </exclusion>
        <exclusion>
          <artifactId>commons-logging</artifactId>
          <groupId>commons-logging</groupId>
        </exclusion>
      </exclusions>
```
See: https://nexus.intern.sda-se.online/repository/sda-se-snapshots/org/sdase/commons/sda-commons-server-kafka/PR-979-SNAPSHOT/sda-commons-server-kafka-PR-979-20210608.122938-2.pom

On the one hand that's a little crazy for all the POMs but on the other hand it's really little effort in our `build.gradle`.

Reply:
This change has already been merged to master separately. 😃